bigchaindb/commands/bigchaindb.py

```diff
@@ -95,21 +95,47 @@ def run_configure(args):
 
 @configure_bigchaindb
 def run_upsert_validator(args):
-    """Store validators which should be synced with Tendermint"""
-
-    b = bigchaindb.BigchainDB()
-    public_key = public_key_from_base64(args.public_key)
-    validator = {'pub_key': {'type': 'ed25519',
-                             'data': public_key},
-                 'power': args.power}
-    validator_update = {'validator': validator,
-                        'update_id': VALIDATOR_UPDATE_ID}
-    try:
-        query.store_validator_update(b.connection, validator_update)
-    except MultipleValidatorOperationError:
-        logger.error('A validator update is pending to be applied. '
-                     'Please re-try after the current update has '
-                     'been processed.')
+    """Initiate and manage elections to change the validator set"""
+
+    b = BigchainDB()
+
+    # Call the function specified by args.action, as defined above
+    globals()[f'run_upsert_validator_{args.action}'](args, b)
+
+
+def run_upsert_validator_new(args, bigchain):
+    """Initiates an election to add/update/remove a validator to an existing BigchainDB network
+
+    :param args: dict
+        args = {
+            'public_key': the public key of the proposed peer, (str)
+            'power': the proposed validator power for the new peer, (str)
+            'node_id': the node_id of the new peer (str)
+            'sk': the path to the private key of the node calling the election (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    :return: election_id (tx_id)
```

Comment:
I don't know how detailed we want to be with doc strings. I like when there is a lot of info, so maybe you can add `:raises:` ?

Reply:
Will do